app/routines/newflow/process_sheerid_webhook_request.rb

```diff
@@ -0,0 +1,30 @@
+module Newflow
+  # When a POST request comes in from SheerID, we:
+  # save the verification id  and
+  class ProcessSheeridWebhookRequest
+    lev_routine
+    uses_routine UpsertSheeridVerification
+
+    protected ###############
+
+    def exec(verification_id:)
+      status.set_job_name(self.class.name)
+      status.set_job_args(verification_id: verification_id)
+
+      verification = upsert_verification(verification_id)
+      if verification.errors.none? && verification.verified? && (user = EmailAddress.verified.find_by(value: verification.email)&.user)
```

Comment:
suggestion...just put the assignment of user = on a different line, and do a `&& user`

Reply:
Considered.